Dockerfile-locust

```diff
@@ -0,0 +1,16 @@
+FROM locustio/locust AS builder
```

Comment:
Can you rename to `Dockerfile.locust` to match the existing pattern `Dockerfile.prow`?